locales/cy/translation.json

```diff
@@ -1120,42 +1120,42 @@
       }
     },
     "pyiTriageDesktopDownload": {
-      "title": "How to prove your identity with the GOV.UK One Login app",
-      "header": "How to prove your identity with the GOV.UK One Login app",
-      "section1": {
-        "circle": "1",
-        "heading": "Download and open the GOV.UK One Login app",
-        "paragraph1": "To download the app, scan the QR code below with your camera. If you already have the app installed, open it now.",
-        "paragraph2": "Sign in to the app, then follow the steps to prove your identity.",
+      "title": "Use the GOV.UK One Login app to prove your identity",
+      "header": "Use the GOV.UK One Login app to prove your identity",
+      "content": {
+        "paragraph1": "The GOV.UK One Login app works by matching your face to your photo ID.",
+        "paragraph2": "First you will need to download, open and use the app. Then you will come back to this page to finish proving your identity.",
+        "subHeading1": "1. Download the GOV.UK One Login app",
+        "paragraph3": "Scan the QR code with your smartphone to download the GOV.UK One Login app.",
+        "warningText": "The app uses a face scan with flashing colours",
+        "warningTextFallback": "Warning",
         "qrCodeAlt": "QR code to download the app",
+        "subHeading2": "Help using the GOV.UK One Login app",
         "details1": {
           "summaryText": "How to scan a QR code",
-          "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur efficitur non quam eget viverra. Duis enim velit, dignissim a feugiat id, ultricies a tortor."
+          "html": "<ol class=\"govuk-list govuk-list--number\"><li>Open the camera app on your phone.</li><li>Position your camera over the QR code.</li><li>Wait until the QR code is recognised by your camera and a link appears on your screen.</li><li>Position your camera over the QR code.</li><li>Tap the link to go to the App Store (for iPhone) or Play Store (for Android).</li><li>Download the GOV.UK One Login app.</li></ol>"
```

Comment:
Prefer to minimise HTML markup in content files as much as possible, especially as we'll want to pass these for Welsh translation. Could we look at moving it into the template and just defining each bullet here?